packages/card/stories/card.stories.ts

```diff
@@ -258,7 +258,7 @@ export const SlottedHeading = (): TemplateResult => {
                     value="Apr 23 Project"
                     quiet
                 ></sp-textfield>
-                <div slot="subheading">LAST MODIFIED ON 6/17/2020, 3:37 PM</div>
+                <div slot="subheading">Last modified on 6/17/2020, 3:37 PM</div>
```

Comment:
This is the change that fails the visual regressions. Bump the cache and you should be good to go.

Reply:
yup -- I was expecting that.